pkg/operator/config.go

```diff
@@ -30,6 +31,15 @@ type Controllers struct {
 	MachineHealthCheck string
 }
 
+type BaremetalControllers struct {
```

Comment:
I think we can remove the "Baremetal" prefix from all of the members of this struct, since it seems redundant with the way we access the values as config.BaremetalControllers.Foo